### ISSUE TITLE
Replace NBN Gateway formatters with NBN Atlas

### ISSUE THREAD

**Issue by JimBacon** [Indicia-Team/drupal-7-module-taxon_formatters#1]
The NBN Gateway has been turned off. A new Atlas has taken its place with all new web services.

**Comment by BirenRathod** [comment]
I found the EasyMap service here -> https://easymap.nbnatlas.org/

**Comment by johnvanbreda** [comment]
There is also a demo of WMS at https://duncanrowland.github.io/NBNMapOverlayExamples/leaflet3.html, though it is very slow.

**Comment by JimBacon** [comment]
Yeah, I know. I wrote the example https://duncanrowland.github.io/NBNMapOverlayExamples/leaflet2.html which may be a quicker technique. Since yesterday the Atlas has really been struggling to respond to its load.
